styles/main.css

```diff
@@ -0,0 +1,7 @@
+img {
+    height: 30px;
+}
+
+.titleError, .authorError, emailError, .phoneError, .isbnError {
```

Comment:
The 'emailError' display appears too early due to a missing selector. Check if the class or ID is targeted correctly.